CMakeLists.txt

```diff
@@ -266,6 +266,8 @@ rocm_setup_version(VERSION ${VERSION_STRING})
 ## Fill in version information for main header file
 configure_file(src/nccl.h.in ${PROJECT_BINARY_DIR}/include/rccl/rccl.h) # For external linking
 configure_file(src/nccl.h.in ${PROJECT_BINARY_DIR}/include/nccl.h)      # Used by some internal files
+file(COPY src/include/rocblas_float8.h DESTINATION ${PROJECT_BINARY_DIR}/include/) # For fp8 support
+file(COPY src/include/rocblas_hip_f8_impl.h DESTINATION ${PROJECT_BINARY_DIR}/include/) # For fp8 support
```

Comment:
Why we need this? Maybe keep aligned with rccl_bfloat16.h?

Reply:
agree. changes to align with rccl_bfloat16.h